NGitLab.Mock/Issue.cs

```diff
@@ -95,5 +95,10 @@ public Models.Issue ToClientIssue()
                 Confidential = Confidential,
             };
         }
+
+        public bool CanUserViewIssue(User user)
+        {
+            return !Confidential || Author.Id == user.Id;
```

Comment:
Should include `Project.CanUserViewConfidentialIssues` so you can actually simplify all calls

Reply:
Good point, hadn't realized that the project object was already referenced inside the issue.